apps/rush-lib/src/schemas/version-policies.schema.json

```diff
@@ -31,6 +31,10 @@
         "nextBump": {
           "description": "Type of next version bump",
           "enum": [ "prerelease", "release", "minor", "patch", "major" ]
+        },
+        "changeLogHostProject": {
```

Comment:
>changeLogHostProject [](start = 9, length = 20)

Should this be something more general like "main project?"

Reply:
+1

---
In reply to: [155397763](https://github.com/Microsoft/web-build-tools/pull/462#discussion_r155397763) [](ancestors = 155397763)

Reply:
That is a good idea.